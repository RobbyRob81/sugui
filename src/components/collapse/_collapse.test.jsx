import * as React from 'react';
import { customRender } from '../../../test/test-utils';
import Collapse from '.';

it('should render without throwing an error', () => {
  const { container } = customRender(<Collapse>Test</Collapse>);
  expect(container.firstChild).toMatchSnapshot();
});

it('should hide content by default', async () => {
  const { getByTestId } = customRender(<Collapse><p>Test</p></Collapse>);
  expect(getByTestId('collapse')).not.toBeVisible();
});

it('should display content if opened is set', () => {
  const { getByTestId } = customRender(<Collapse expanded><p>Test</p></Collapse>);
  expect(getByTestId('collapse')).toBeVisible();
});
