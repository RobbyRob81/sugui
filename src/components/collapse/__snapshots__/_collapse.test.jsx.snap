// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without throwing an error 1`] = `
<div
  class="sc-bdVaJa bhruBH"
  data-testid="collapse"
  style="height: 0px; opacity: 0;"
>
  Test
</div>
`;
